demo/demo.py

```diff
@@ -65,9 +75,14 @@ def get_output(video_path,
         fps (int): Number of picture frames to read per second. Default: 30.
         font_size (int): Font size of the label. Default: 20.
         font_color (str): Font color of the label. Default: 'white'.
-        resize_algorithm (str): The algorithm used for resizing.
-            Default: 'bicubic'. For more information,
-            see https://ffmpeg.org/ffmpeg-scaler.html.
+        target_resolution (None | tuple[int, int]): Same as that in
+            ``VideoFileClip``. Set to (desired_height, desired_width) to have
```

Comment:
resolutions are often in the order of width, height. e.g. ffmpeg uses that order.

Reply:
refer to [this](https://github.com/Zulko/moviepy/blob/master/moviepy/video/io/VideoFileClip.py#L38-L42), and it revert the resolution [here](https://github.com/Zulko/moviepy/blob/2cd208c3791931aaea21b0ae278469bdfa57a96d/moviepy/video/io/ffmpeg_reader.py#L40-L41)

Reply:
we do not need to expose the order of `VideoFileClip` here. moviepy is an implementation detail, that can be replaced at any time

Reply:
considering moviepy has installation issues both on windows and unix, there is a great probability that we replace it with opencv

Reply:
It will be an alternative for future plan.